app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    team_total_rating = sum([player.get_rating() for player in players])
```

Comment:
```suggestion
    team_total_rating = sum(player.get_rating() for player in players)
```
No need to convert to list)

Reply:
sry, my fault